Formula/apache-httpd.rb

```diff
@@ -0,0 +1,128 @@
+class ApacheHttpd < Formula
+  desc "Apache HTTP server"
+  homepage "https://httpd.apache.org/"
+  url "https://www.apache.org/dyn/closer.cgi?path=httpd/httpd-2.4.27.tar.bz2"
+  sha256 "71fcc128238a690515bd8174d5330a5309161ef314a326ae45c7c15ed139c13a"
+
+  bottle do
+    sha256 "d35e67ae745053cda23273a520ead6bde506ec8082057edcd3185c9e36eae483" => :sierra
+    sha256 "4ea7056d6c08f9cf49507869aa8c78e3522c601d87b3347d31561ba05b33d41c" => :el_capitan
+    sha256 "e6ae5c4d38b40bcf3d85877ac117a1ea048de774edec3c2b0ff714bc01a6d40a" => :yosemite
+  end
+
+  deprecated_option "with-http2" => "with-nghttp2"
+
+  depends_on "apr"
+  depends_on "apr-util"
+  depends_on "openssl"
+  depends_on "pcre"
+
+  depends_on "nghttp2" => :optional
+
+  def install
+    # point config files to opt_prefix instead of the version-specific prefix
+    inreplace "Makefile.in",
+      '#@@ServerRoot@@#$(prefix)#', '#@@ServerRoot@@'"##{opt_prefix}#"
+
+    args = %W[
+      --enable-mods-shared=all
+      --enable-unique-id
+      --enable-ssl
+      --enable-dav
+      --enable-cache
+      --enable-logio
+      --enable-deflate
+      --enable-cgi
+      --enable-cgid
+      --enable-suexec
+      --enable-rewrite
```

Comment:
Should probably include `--enable-pie`.

Reply:
Why?

Reply:
Building a position-independent executable is good for security of the binary from certain attacks. It's generally recommended by many distros.